### ISSUE TITLE
Can't place liquidity orders

### ISSUE THREAD

**Issue by alfetopito** [gnosis/dex-react#1235]
# Can't place liquidity orders

On #1234 

https://pr1234--dexreact.review.gnosisdev.com/liquidity

## Steps to reproduce

1. Select any token combination
2. Input any spread
3. Click on `submit transaction`

## Expected

Liquidity orders are placed

## Observed

Tx is not placed. Error shown on the UI and logged to the console:
![screenshot_2020-07-16_09-50-33](https://user-images.githubusercontent.com/43217/87699832-48234600-c74a-11ea-8b8b-766a3cb1580e.png)

## Details:
- Firefox 78.0.2 (64-bit) AND Chrome(ium) Version 83.0.4103.116 (Official Build) Arch Linux (64-bit)
- Metamask 8.0.4
- ~No issues on Chrome, same MM version.~
- No issues on FF with WallectConnect.

**Comment by alfetopito** [comment]
Bisect result:

```
~/work/dfusion/dex-react(9df2dc46) » git bisect good
af436d534cc3c3adc92342378a6733831a49cf15 is the first bad commit
commit af436d534cc3c3adc92342378a6733831a49cf15
Author: Anxo Rodriguez <anxolin@gmail.com>
Date:   Tue Jul 7 21:55:07 2020 +0200

    Fix trust gas issue (#1170)
    ...

```

**Comment by alfetopito** [comment]
As far as I can tell, the issue might be on the estimated gas price:
![screenshot_2020-07-16_14-43-55](https://user-images.githubusercontent.com/43217/87725740-b4b23b00-c772-11ea-9f24-29f027f7a4f3.png)
== 1000000001 on base 10

Couldn't find out much more than that
